libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -929,16 +923,16 @@ fn parse_amount(amount_str: String) -> Result<pb::Amount> {
     Ok(pb::Amount { unit: Some(unit) })
 }
 
-impl From<pb::Channel> for crate::models::Channel {
+impl From<pb::Channel> for Channel {
     fn from(c: pb::Channel) -> Self {
         let state = match c.state.as_str() {
-            "OPENINGD" | "CHANNELD_AWAITING_LOCKIN" => crate::models::ChannelState::PendingOpen,
-            "CHANNELD_NORMAL" => crate::models::ChannelState::Opened,
-            "CLOSED" => crate::models::ChannelState::Closed,
-            _ => crate::models::ChannelState::PendingClose,
+            "OPENINGD" | "CHANNELD_AWAITING_LOCKIN" => ChannelState::PendingOpen,
+            "CHANNELD_NORMAL" => ChannelState::Opened,
+            "ONCHAIN" | "CLOSED" => ChannelState::Closed,
```

Comment:
According to core lightning docs there is no status CLOSED https://docs.corelightning.org/reference/lightning-listpeers

Reply:
If I search that link for `"CLOSED"` I find:

> The objects in the channels array will have at least these fields:
> - state: Any of these strings: 
>   ...
>   -  "CLOSED": The channel closure has been confirmed deeply. The channel will eventually be removed from this array.

Is this not the same status we're interested in?

Reply:
@ok300 I agree

Reply:
Right .  My miss.